### DISCUSSION TITLE
(root) Additional property payload is not allowed

### DISCUSSION THREAD

**Discussion by t3rminalx** [stashapp/stash#4328] [Q&A] [answered]
I've tried to run the sample docker-compose yml file but get the above error.

The only time I see root mentioned is on line 32 but if I change that it'll break Stash i assume.

**Comment by WithoutPants** [comment, marked as answer]
There's something wrong with the format of your docker-compose file. See https://forums.docker.com/t/docker-compose-additional-property-is-not-allowed/131767
